### PR TITLE
added basic setup for new CRUD APIs

### PR DESCRIPTION
**Pull request by hongmm**
@jase-d @ClaySamuelC @mbbph

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -8,40 +9,68 @@ const cors = require('cors');
 const app = express();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: true}));
-app.use(express.static(__dirname + '/../client/dist'))
+app.use(express.static(path.join(__dirname, '..', 'client', 'dist')))
```
Comment (ClaySamuelC) [inline server/app.js:12]: missing semicolon here

File: server/app.js [hunk 2]
```diff
@@ -8,40 +9,68 @@ const cors = require('cors');
...
+app.use('/listing', express.static(path.join(__dirname, '..', 'client', 'dist')));
 app.use(cors());
 
-app.get('/api/carousel', (req, res) => {
-  Product.find({}, (err, products) => {
+// Need to add in rest of the CRUD verbs
+app.get('/listing/*', (req, res) => { res.sendFile(path.join(__dirname, '..', 'client', 'dist', 'index.html')); });
```
Comment (ClaySamuelC) [inline server/app.js:17]: This is good, but I would recommend splitting this into multiple lines to make it more clear to glance value

**Review by ClaySamuelC — APPROVED** [review]
Looks good, I made a couple comments that are not necessarily urgent.
Mostly just visual things, I would add semicolons to lines 12, 31, 32, 36, 37, 41, and 42 in server/app.js.
I also recommend putting todo comments in places that you plan to update (the bodies of your CRUD requests).
These are all just things that reduce technical debt, other than that it looks great.

**Comment by hongmm** [comment]
Thanks for the feedback! I've added in the changes for the most part, for the rest I will add in a future PR.

**Pull request merged**